types/block.go

```diff
@@ -54,6 +54,9 @@ type Block struct {
 	// a challenge
 	Proof PoStProof `json:"proof"`
 
+	// A Unix Timestamp generated at block creation.
```

Comment:
"The timestamp, in seconds since the Unix epoch, at which this block was created"